src/cs2_sdk/interfaces/cgameresourceserviceserver.h

```diff
@@ -20,15 +20,17 @@
 #pragma once
 #include <platform.h>
 #include "interfaces/interfaces.h"
-#include "../../addresses.h"
+#include "../../gameconfig.h"
 
 class CGameEntitySystem;
 
+extern CGameConfig *g_GameConfig;
+
 class CGameResourceService
 {
 public:
 	CGameEntitySystem *GetGameEntitySystem()
 	{
-		return *reinterpret_cast<CGameEntitySystem **>((uintptr_t)(this) + offsets::GameEntitySystem);
+		return *reinterpret_cast<CGameEntitySystem **>((uintptr_t)(this) + g_GameConfig->GetOffset("GameEntitySystem"));
```

Comment:
GetOffset should only be run once here.

Reply:
See https://github.com/Source2ZE/CS2Fixes/pull/36#discussion_r1361046665